### ISSUE TITLE
请问一下，中断训练后如何继续训练。

### ISSUE THREAD

**Issue by kabrate** [yzyouzhang/AIR-ASVspoof#12]
您好，请问一下比如训练了100个epoch，如何在已经训练权重的基础上继续训练呢？

**Comment by yzyouzhang** [comment]
您好，感谢你的关于继续训练的提问。
现在的代码可以支持继续训练了，请在继续训练的时候运行train.py的时候加上--continue training。记住要同时调整学习率，因为在训练过程中学习率有阶段性调整。谢谢！
Hello, thanks for your question about continuing training.
Our code can now support to continue training. Please use the argument of "--continue_training" when you run train.py the second time. Remember to reset the learning rate since we adjust the learning rate during training based on epochs.

**Comment by beijita-yegucheng** [comment]
您好！为什么我在继续训练的时候，输入了python train.py --add_loss ocsoftmax -o ./models/ocsoftmax --gpu 0 --continue_training。可是程序自动将之前的文件删除了，以至于没办法继续训练，能不能，具体说一下，当训练停止后怎么继续训练？十分感谢！！！

**Comment by yzyouzhang** [comment]
已修复！感谢指正！
